README.rst

```diff
@@ -36,109 +36,96 @@ your Python code to a file.
 Make sure that you look at the `User Resources`_ section as well for links
 to documentation and larger examples.
 
-Usage Examples
+Usage
 --------------
 
-To run these minimal examples, run the Python3 interpreter from
-the terminal using the ``python3`` command:
+To start out, you'll need a way to work with Python. We recommend the
+`ev3dev Visual Studio Code extension`_. If you're interested in using that,
+check out our `Python + VSCode introduction tutorial`_ and then come back
+once you have that set up.
 
-.. code-block:: bash
+Otherwise, you can can work with files via an SSH connection with an editor
+such as `nano`_, use the Python interactive REPL (type ``python3``), or roll
+your own solution.
+
+The template for a Python script
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every Python program should have a few basic parts. Use this template
+to get started:
 
-  $ python3
-  Python 3.4.2 (default, Oct  8 2014, 14:47:30)
-  [GCC 4.9.1] on linux
-  Type "help", "copyright", "credits" or "license" for more information.
-  >>>
+.. code-block:: python
 
-The ``>>>`` characters are the default prompt for Python. In the examples
-below, we have removed these characters so it's easier to cut and
-paste the code into your session.
+   #!/usr/bin/env python3
+   from ev3dev2.motor import LargeMotor, OUTPUT_A
+   from ev3dev2.sensor.lego import TouchSensor
+   from ev3dev2.led import Leds
 
-Required: Import the library
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   # TODO: Add code here
 
-If you are using an EV3 brick (which is the case for most users), add the
-following to the top of your file:
+The first line should be included in every Python program you write
+for ev3dev. It allows you to run this program from Brickman, the graphical
+menu that you see on the device screen. You will need to add additional classes
+to the import list if you want to use other types of devices.
 
-.. code-block:: python
+You should use the ``.py`` extension for your file, e.g. ``my-file.py``.
 
-  import ev3dev.ev3 as ev3
+Important: Make your script executable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you are using a BrickPi, use this line:
+To be able to run your Python file, **your program must be executable**. If
+you're using the `ev3dev Visual Studio Code extension`_, this step will be
```

Comment:
'you are'

Reply:
I think "If you're using the `ev3dev Visual Studio Code extension`_, **you can skip this step**, since it will be automatically performed when you download your code to the brick." sounds better. Otherwise it sounds like you can skip this step in any case.